### ISSUE TITLE
Detox - Rewrite tests now that demo mode doesn't show the in-app review popup

### ISSUE THREAD

**Issue by TKDickson** [department-of-veterans-affairs/va-mobile-app#7159]
A nonzero number of detox tests are employing workarounds for the in-app review popup (I know for sure letters, from #6930, but Rachael mentioned she had done workarounds elsewhere as well)

Once ticket #7158 is finished, we should remove the workaround(s) so the tests run faster.

**Comment by TKDickson** [comment]
Ended up being larger than anticipated - some of that trying, and failing, to remove the reset workaround from the AF script. Separated that into a new ticket
